### PR TITLE
Fix phpunit-wrapper from resetting variables in global scope in WrapperRunner

### PR DESCRIPTION
**Pull request by proggga**
phpunit-wrapper has a problem

it's broken in WrapperRunner
phpunit reset global variables between runs
https://phpunit.readthedocs.io/en/9.3/fixtures.html#global-state

I've got problems with global state in phpunit and this fix helped me.
I tried to user WrapperRunner in my big project, and this helps

this fix prevent variables in phpwrapper (they all in global scope) from turning to null 

(Also I removed assert here like in #539, I need this for passing all tests)

### CODE REVIEW

File: src/Runners/PHPUnit/Runner.php [hunk 1]
```diff
@@ -65,8 +65,6 @@ private function fillRunQueue(array $availableTokens): void
             && ($token = array_shift($availableTokens)) !== null
         ) {
             $executebleTest = array_shift($this->pending);
-            /** @psalm-suppress RedundantConditionGivenDocblockType **/
-            assert($executebleTest !== null);
```
Comment (Slamdunk) [inline src/Runners/PHPUnit/Runner.php:69]: If you leave this here, what test is failing?

**Review by Slamdunk — CHANGES_REQUESTED** [review]
I get the idea of the issue, but please can you try to write a test for the bug?
It seems an important one, I'd like to prevent it from being reintroduced by a future refactor

File: src/Runners/PHPUnit/Runner.php (hunk 1 above)
Comment (proggga) [reply src/Runners/PHPUnit/Runner.php:69]: Please look at  #539, I wrote there more info about problem

**Comment by proggga** [comment]
> I get the idea of the issue, but please can you try to write a test for the bug?
> It seems an important one, I'd like to prevent it from being reintroduced by a future refactor

It's problematic, cause phpunit-wrapper fails, and paratest doing nothing, he is waiting for some command or idk what. It will never complete.

File: src/Runners/PHPUnit/Runner.php (hunk 1 above)
Comment (Slamdunk) [reply src/Runners/PHPUnit/Runner.php:69]: Thank you for bringing this into my attention.
I've pushed to master the change, so now you cane rebase the PR onto it

**Comment by Slamdunk** [comment]
> It's problematic, cause phpunit-wrapper fails, and paratest doing nothing, he is waiting for some command or idk what. It will never complete.

That's ok: the pipeline has `timeout-minutes: 3` set, so it will fail is a test is hanging forever.

Please try to write a test so I can see the pipeline failing

**Comment by proggga** [comment]
> > It's problematic, cause phpunit-wrapper fails, and paratest doing nothing, he is waiting for some command or idk what. It will never complete.
> 
> That's ok: the pipeline has `timeout-minutes: 3` set, so it will fail if a test is hanging forever.
> 
> Please try to write a test so I can see the pipeline failing

Okey, I written test, that fails on master branch (branch without my fix)
problem was in param in phpunit "backupGlobal=true", you can check this out

File: composer.json [hunk 2]
```diff
@@ -37,14 +37,14 @@
         "ekino/phpstan-banned-code": "^0.3.1",
         "ergebnis/phpstan-rules": "^0.15.2",
         "infection/infection": "^0.17.3",
-        "phpstan/phpstan": "^0.12.45",
+        "phpstan/phpstan": "^0.12.49",
```
Comment (Slamdunk) [inline composer.json:40]: The [master branch in your fork](https://github.com/proggga/paratest/tree/master) is way outdated: you should pull the latest commits from upstream, and then rebase the branch onto it, so your PRs don't get polluted by this kind of changes

**Comment by proggga** [comment]
Also pls add label after all "hacktoberfest-accepted"

File: composer.json (hunk 2 above)
Comment (proggga) [reply composer.json]: fixed

**Review by Slamdunk — APPROVED** [review]

**Comment by Slamdunk** [comment]
Thank you @proggga

**Pull request merged**